packages/slice-machine/components/ReviewModal/index.tsx

```diff
@@ -113,8 +114,7 @@ const ReviewModal: React.FunctionComponent<ReviewModalProps> = () => {
   ): Promise<void> => {
     try {
       startLoadingReview();
-      // env should never be undefined or null
```

Comment:
what about the tracker?

Reply:
not in this PR, but in this PR https://github.com/prismicio/slice-machine/pull/283, in the new tracker implementation I suggest something that get ride of this condition :)

Reply:
neet